src/components/ErrorNotification/index.js

```diff
@@ -0,0 +1 @@
+export * from './ErrorNotification';
```

Comment:
Remove this file 
<img width="552" alt="image" src="https://github.com/user-attachments/assets/6358a34f-617c-4eab-938c-2128aab2d58b" />